subprojects/core-api/src/main/java/org/gradle/api/file/DuplicatesStrategy.java

```diff
@@ -54,7 +54,7 @@ public enum DuplicatesStrategy {
 
     /**
      * The default strategy, which is to inherit the strategy from the parent copy spec, if any,
-     * or {@link DuplicatesStrategy#INCLUDE} if the copy spec has no parent.
+     * or {@link DuplicatesStrategy#FAIL} if the copy spec has no parent.
```

Comment:
💅 I'm not convinced this is a good place to put a default value, but that is for some platform owner to decide.

Reply:
Fixed to be clean and simple.